electron/utils.ts

```diff
@@ -404,6 +431,10 @@ const writeGameconfig = async (game: string) => {
 }
 
 async function checkForUpdates() {
```

Comment:
Wasn't sure where to put this comment. But maybe on startup show a dialog or something, that the user appears to be offline. So some functionality might not work.

Reply:
Agreed! A message on startup could fix this.

Reply:
Okay! Will try to implement. Depending on complexity, might have to become a follow-up PR.